python/rmm/_lib/device_buffer.pyx

```diff
@@ -214,8 +214,8 @@ cpdef void copy_ptr_to_host(uintptr_t db,
     Note
     ----
 
-    If ``stream`` is the default stream, we synchronize on it.
-    However if a non-default ``stream`` is provided, we leave
+    If ``stream`` is the default stream, it is synchronized after the copy.
+    However if a non-default ``stream`` is provided, leave
```

Comment:
```suggestion
    However if a non-default ``stream`` is provided, it is up to the user to synchronize.
```

Reply:
Not sure why GitHub wouldn't let me commit this as-is. Anyways copied it and committed it in the latest change. Please take a look 🙂